Core/src/main/java/com/plotsquared/core/plot/Plot.java

```diff
@@ -1483,7 +1483,7 @@ private Location toHomeLocation(Location bottom, BlockLoc relativeHome) {
      */
     public void setHome(BlockLoc location) {
         Plot plot = this.getBasePlot(false);
-        if (BlockLoc.ZERO.equals(location) || BlockLoc.MINY.equals(location)) {
+        if (location != null && (BlockLoc.ZERO.equals(location) || BlockLoc.MINY.equals(location))) {
```

Comment:
The null check should not be required as we don't call equals on the nullable object itself